config/initializers/devise.rb

```diff
@@ -1,4 +1,33 @@
 # frozen_string_literal: true
+#
+class TurboFailureApp < Devise::FailureApp
```

Comment:
So I watched the GoRails video, and I think it's out of date already at this point unfortunately. I think we can actually skip over all of this custom controller approach and configuration changes in favor of a much simpler solution...